content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,229 @@
++++
+title = "Kubeflow Versioning Policies"
+description = "Versioning policies and status of Kubeflow applications and other components"
+weight = 05
++++
+
+This page describes the Kubeflow versioning policies and provides a version 
+matrix for Kubeflow applications and other components.
+
+## Kubeflow versioning
+
+Kubeflow version numbers are of the form **vX.Y.Z**, where **X** is the major 
+version, **Y** is the minor version, and **Z** is the patch version. The
+versioning policy follows the [Semantic Versioning](https://semver.org/) 
+terminology.
+
+The version name **vX.Y.Z** refers to the version (git tag) of the 
+[kfctl release](https://github.com/kubeflow/kubeflow/releases). 
+If the version number includes an appendix **-rcN**, where **N** is a
+number, the appendix indicates a *release candidate*, which is a pre-release 
+version of an upcoming release.
+
+Examples of Kubeflow version numbers:
+
+* `v0.7.0`
+* `v0.7.0-rc8`
+* `v1.0.0`
+* `v1.0.1`
+
+<a id="app-versioning"></a>
+## Application versioning and stable status
+
+Starting from the release of Kubeflow v1.0, the Kubeflow community 
+attributes *stable status* to those applications and components that 
+meet a defined level of stability, supportability, and upgradability.
+
+When you deploy Kubeflow to a Kubernetes cluster, your deployment includes a
+number of applications. Application versioning is independent of Kubeflow 
+versioning. An application moves to version 1.0 when the application meets 
+certain 
+[criteria](https://github.com/kubeflow/community/blob/master/guidelines/application_requirements.md) 
+in terms of stability, upgradability, and the provision of services such as 
+logging and monitoring. 
+
+When an application moves to version 1.0, the Kubeflow community will 
+decide whether to include that version of the application in the next major 
+or minor release of Kubeflow.
+
+<a id="application-matrix"></a>
+## Kubeflow application matrix
+
+The following table shows the stable status of the applications that
```

Comment:
Thanks! I've followed your first suggestion, but I've left the second one as passive voice, because most people will use a standard, predefined manifest. It's important that they realise that whatever's in the manifest is what gets deployed.

FYI, I had a community review meeting of this doc earlier today, and as a result I've added a bit that states that the kfctl tool doesn't discriminate based on app status.